### ISSUE TITLE
Django REST Framework (DRF) integration error

### ISSUE THREAD

**Issue by grzegorzbialy** [barseghyanartur/django-fobi#62]
If you try to get form using /fobi-form-entry/*slug*/ there is a TypeError:
```
get_custom_field_instances() missing 1 required positional argument: 'integrate_with'
Exception Location: /fobi/base.py in _get_custom_field_instances, line 1620
```

It seems that integrate_with parameter is missing when calling get_custom_field_instances() in fobi.base, line 1620. If you add following it works:

```
custom_field_instances = self.get_custom_field_instances(
    integrate_with=integrate_with,  # added here
    request=request,
    form_entry=form_entry,
    form_element_entries=form_element_entries,
    has_value=has_value,
    **kwargs
)
```

**Comment by barseghyanartur** [comment]
@grzegorzbialy:

Thanks for reporting this. I'll fix it shortly.

Officially only LIST, OPTIONS and PUT make sense and are supported. Since you're not supposed to use GET action, I haven't covered it in [tests](https://github.com/barseghyanartur/django-fobi/blob/master/src/fobi/tests/test_drf_integration.py), although it seems to work fine in the browsable API on [Heroku demo](https://django-fobi.herokuapp.com/api/fobi-form-entry/). 

But, in any case, such nasty errors shouldn't happen. Thanks again for bringing this up.

**Comment by grzegorzbialy** [comment]
@barseghyanartur Thank you for quick reply.

I see it's working on demo site using GET: https://django-fobi.herokuapp.com/api/fobi-form-entry/test-form-2/
I think it's because DEBUG is set to True on demo app and fobi passes integrate_with when DEBUG is True.

base.py, line 1604:
https://github.com/barseghyanartur/django-fobi/blob/master/src/fobi/base.py#L1604)

**Comment by barseghyanartur** [comment]
@grzegorzbialy:

Ah, I see - it's an obvious bug. :) Gonna be fixed today. I think I'll add some GET (`retrieve`) tests ass well. 

Thanks again for digging in.

**Comment by barseghyanartur** [comment]
Solved in [0.11.6](https://pypi.python.org/pypi/django-fobi/0.11.6).
